tests/Contrib/Unit/OTLPGrpcExporterTest.php

```diff
@@ -4,15 +4,104 @@
+                                ->andReturn(
+                                    [
+                                        'unused response data',
+                                        new class() {
+                                            public $code;
+                        
+                                            public function __construct()
+                                            {
+                                                $this->code = \Grpc\STATUS_OK;
+                                            }
+                                        },
+                                    ]
+                                )
+                                ->getMock()
+                        )
+                        ->getMock();
+        $exporter = new Exporter(
+            //These first parameters were copied from the constructor's default values
+            'localhost:4317',
+            true,
+            '',
+            '',
+            false,
+            10,
+            $mockClient
+        );
+               
+        $exporterStatusCode = $exporter->export([new SpanData()]);
+
+        $this->assertEquals(Exporter::SUCCESS, $exporterStatusCode);
+    }
+
+    public function testExporterUnexpectedGrpcResponseStatus()
+    {
+        /** @var MockInterface&TraceServiceClient */
+        $mockClient = Mockery::mock(TraceServiceClient::class)
+                        ->shouldReceive('Export')
+                        ->withArgs(function ($request) {
+                            return (count($request->getResourceSpans()) == 1);
+                        })
+                        ->andReturn(
+                            Mockery::mock(UnaryCall::class)
+                                ->shouldReceive('wait')
+                                ->andReturn(
+                                    [
+                                        'unused response data',
+                                        new class() {
+                                            public $code;
+                        
+                                            public function __construct()
+                                            {
+                                                $this->code = 'An unexpected status';
+                                            }
+                                        },
+                                    ]
+                                )
+                                ->getMock()
+                        )
+                        ->getMock();
+        $exporter = new Exporter(
+            //These first parameters were copied from the constructor's default values
+            'localhost:4317',
+            true,
+            '',
+            '',
+            false,
+            10,
+            $mockClient
+        );
+               
+        $exporterStatusCode = $exporter->export([new SpanData()]);
+
+        $this->assertEquals(Exporter::FAILED_NOT_RETRYABLE, $exporterStatusCode);
```

Comment:
```suggestion
        $this->assertSame(Exporter::FAILED_NOT_RETRYABLE, $exporterStatusCode);
```

Minor, but is more ideal to use `assertSame`, otherwise things like `'1' == 1` would not be detected.

Reply:
That's super good to know! Went ahead and updated them